app/views/claims_closed.scala.html

```diff
@@ -29,11 +29,9 @@
 @(model: ClosedClaimListViewModel,
   caseStatusHints: DropdownHints,
   searchForm: Form[String],
-  postAction: Call,
-  companyName: String,
-  eori: String)(implicit request: IdentifierRequest[_], messages: Messages, appConfig: AppConfig)
+  postAction: Call)(implicit request: IdentifierRequest[_], messages: Messages, appConfig: AppConfig)
 
-@key = @{"claim-list-closed"}
+@key = @{"claim.list.closed"}
```

Comment:
I think we have kept `-` everywhere else

Reply:
No we haven't @haseemtasheel. This repo uses dots everywhere so we should stay consistent with that pattern. If we need to change it to hyphens like in CDSR frontend, then we should create a separate (tech-debt) ticket for it.

Reply:
We have changed the style to reflect the style used in CDS-reimbursement, so the dashes are in the key, we then use the key on the view